web/ascii_art_web/templates/img-upload.html

```diff
@@ -5,10 +5,26 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>image to ascii converter</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
```

Comment:
Unclear what this is for?

Reply:
that's the bootstrap css linking to the cdn

Reply:
I understand what the links do. I was asking why you include Bootstrap at all? It's quite a big file, and it didn't seem like you were using it.

But after another look at the html, I see you reference its classes. Sorry.